hypothesis-python/tests/ghostwriter/test_ghostwriter.py

```diff
@@ -255,3 +256,7 @@ def g(arg: float) -> None:
 
     source_code = ghostwriter.equivalent(f, g)
     assert "arg=st.one_of(st.integers(), st.floats())" in source_code
+
+
+def test_module_with_mock_works():
```

Comment:
Yes, it used to - the validation would pass because it checks various attributes, but then the actual machinery failed because mocks aren't really callable functions.  Commented.